crates/fluvio-spu/src/control_plane/dispatcher.rs

```diff
@@ -131,6 +136,7 @@ impl ScDispatcher<FileReplica> {
 
                 _ = status_timer.next() =>  {
                     self.send_status_back_to_sc(&mut sink).await?;
+                    self.send_mirror_status_back_to_sc(&mut sink).await?;
```

Comment:
why sending status twice?  Should consolidate into a single status

Reply:
`send_status_back_to_sc` is only sending lrs status. 
I renamed this function to `send_lrs_status_back_to_sc` to make more sense.